torch/fx/experimental/migrate_gradual_types/constraint_generator.py

```diff
@@ -0,0 +1,334 @@
+import torch
+import operator
+from typing import Callable, Dict
+from torch.fx.experimental.migrate_gradual_types.constraint import ApplyBroadcasting, CalcProduct  # type: ignore[import]
```

Comment:
Why all the `type: ignore`? Is lint complaining? Seems like something is broken if so, because `constraint.py` is there, right?

Reply:
Fixed. The __init__.py file was missing.